build-system/tasks/visual-diff/index.js

```diff
@@ -372,8 +399,10 @@ async function generateSnapshots(percy, webpages) {
     await percy.snapshot('Blank page', page, SNAPSHOT_EMPTY_BUILD_OPTIONS);
   }
 
+  applyAmpConfig();
   log('verbose', 'Generating snapshots...');
   await snapshotWebpages(percy, browser, webpages);
+  cleanupAmpConfig();
```

Comment:
I don't think this is a good idea any more, since tests that are run after this will no longer see `AMP_CONFIG` in the runtime. I think you can completely eliminate the `cleanupAmpConfig()` step.

(In the past, it was necessary because we were switching between `canary` and `prod`.)

Reply:
Done